OpenRA.Mods.Common/Widgets/Logic/Ingame/DebugLogic.cs

```diff
@@ -0,0 +1,48 @@
+﻿#region Copyright & License Information
+/*
+ * Copyright 2007-2019 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using OpenRA.Graphics;
+using OpenRA.Mods.Common.Traits;
+using OpenRA.Network;
+using OpenRA.Widgets;
+
+namespace OpenRA.Mods.Common.Widgets.Logic.Ingame
+{
+	public class DebugLogic : ChromeLogic
+	{
+		[ObjectCreator.UseCtor]
+		public DebugLogic(Widget widget, OrderManager orderManager, World world, WorldRenderer worldRenderer)
+		{
+			var worldWorldActor = world.WorldActor;
```

Comment:
Since it's only called once and we don't really save much in terms of line length, I think this `var` can be dropped.